python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -170,6 +170,8 @@ class WrappedTransformerImageClassificationModel(object):
 
     def __init__(self, model):
         """Initialize the WrappedTransformerImageClassificationModel."""
+        device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
This is fine as a default, but I wonder if there is an extra parameter we can add to allow user to specify the device as well?  That way, we would explicitly fail if we don't support the option they set, or if they have a GPU but run into an issue where they still have to use CPU we give them that option still.

Reply:
I don't think you should add that parameter as part of this PR though, this can be merged as is probably.

Reply:
noted! will add this to the backlog